src/main/java/io/cdap/plugin/salesforce/plugin/source/batch/SalesforceMultiSourceConfig.java

```diff
@@ -104,9 +104,11 @@ public String getSObjectNameField() {
     return Strings.isNullOrEmpty(sObjectNameField) ? SOBJECT_NAME_FIELD_DEFAULT : sObjectNameField;
   }
 
-  @Override
```

Comment:
why was this removed?
also, remove newline added below

Reply:
Previously it was extending the class, which was containing validate method. But now the validate method has been moved to ConnecterConfig and this class is not extending ConnecterConfig. That’s why this annotation has been removed.